pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -806,3 +846,62 @@ func StartStaleKeysReclaimer(ctx context.Context) {
 		}
 	}()
 }
+
+func IPsecDefaultDropPolicy(ipv6 bool) error {
+	defaultDropPolicy := defaultDropPolicyIPv4
+	family := netlink.FAMILY_V4
+	if ipv6 {
+		defaultDropPolicy = defaultDropPolicyIPv6
+		family = netlink.FAMILY_V6
+	}
+
+	err := netlink.XfrmPolicyUpdate(defaultDropPolicy)
+
+	// We move the existing XFRM OUT policy to a lower priority to allow the
+	// new priorities to take precedence.
+	// This code can be removed in Cilium v1.15 to instead remove the old XFRM
+	// OUT policy and state.
+	removeStaleXFRMOnce.Do(func() {
+		deprioritizeOldOutPolicy(family)
+	})
+
+	return err
+}
+
+func deprioritizeOldOutPolicy(family int) {
+	policies, err := netlink.XfrmPolicyList(family)
+	if err != nil {
+		log.WithError(err).Error("Cannot get XFRM policies")
+	}
+	for _, p := range policies {
+		if p.Dir == netlink.XFRM_DIR_OUT && p.Mark.Mask == linux_defaults.IPsecMarkMaskOut {
+			p.Priority = oldXFRMOutPolicyPriority
+			if err := netlink.XfrmPolicyUpdate(&p); err != nil {
+				log.WithError(err).WithFields(logrus.Fields{
+					logfields.SourceCIDR:      p.Src,
+					logfields.DestinationCIDR: p.Dst,
+				}).Error("Failed to deprioritize old XFRM out policy")
+			}
+		}
+		// draft for test
+		if p.Dir == netlink.XFRM_DIR_IN {
```

Comment:
Don't need to deprioritize policy for dir-in and dir-fwd, I did this just for test.